brayns/common/types.h

```diff
@@ -335,6 +340,9 @@ enum class CameraMode
     inspect
 };
 
+// A clip plane is defined by a normal and a distance to the center of the scene bounding box
+typedef std::vector< Vector4f > ClipPlanes;
```

Comment:
That's not my point. Both Livre and RTNeuron use 4 floats also. My concern is about the specification of what the 4th component means. In the usual plane equation the plane is defined by a b c d, and the points on the plane are those for which ax + by + cz + d = 0, in this equation a, b, c is a vector normal to the plane.
Your 4th component on the other hand is bizarre, because there's no way of writing a similar equation to the same before without knowing the bounding box of the scene, and that's something that changes, so your plane is not fully defined is it's not in the context of a specific scene, which is bad.